wgsl/index.bs

```diff
@@ -3843,42 +3846,23 @@ Each pipeline input or output is one of:
 A <dfn noexport>built-in input variable</dfn> provides access to system-generated control information.
 The set of built-in inputs are listed in [[#builtin-variables]].
 
-To declare a variable for accessing a particular input built-in *X*:
+To declare a variable for accessing a particular input built-in *X* from an entry point:
 
-* Declare a module-scope variable in the [=storage classes/in=] storage class,
+* Declare a parameter of the entry point function,
```

Comment:
`parameter of` or `parameter to`? Not sure ....

Reply:
uh, not sure either :|